### PR TITLE
Adds base and builder image for alpine node 10.13.0

### PR DESCRIPTION
**Pull request by brandondoran**
Add new Alpine images for Node 10.13.0. This is actually using 10.12.0 because 10.13.0 is brand new and the official images hasn't been pushed to docker hub yet: https://hub.docker.com/_/node/.

### CODE REVIEW

File: 10.13.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM node:10.12.0-alpine
```
Comment (visormatt) [inline 10.13.0/Dockerfile]: Should this be from `node:10.13.0-alpine` https://goo.gl/McJeVi

Comment (visormatt) [reply 10.13.0/Dockerfile]: nvm, just needed to read the PR :)

Comment (brandondoran) [reply 10.13.0/Dockerfile]: 🔥 10.13.0, now hot off the presses 🔥

Comment (visormatt) [reply 10.13.0/Dockerfile]: It's pretty funny, I was just looking at react 16.7 as well, talk about riding the edge 💯

Comment (brandondoran) [reply 10.13.0/Dockerfile]: Haha, yeah. Node 10.13 just became LTS the before yesterday and I've been refreshing the docker hub page so I could upgrade to it all day.

**Pull request merged**